apps/music/js/ui/title_bar.js

```diff
@@ -61,13 +65,18 @@ var TitleBar = {
 
   showScanProgress: function(info) {
     this.scanProgress.classList.remove('hidden');
+    this.scanSpinner.classList.remove('hidden');
```

Comment:
Nit: could you add a comment saying that this is part of the workaround? It'll make it easier to remove this when the Gecko fix lands.